bloom-housing/bloom
# fix: lottery - history log reflecting all changes to a listing

This PR addresses [#(4310)](https://app.zenhub.com/workspaces/bloom-5dc32d7144bd400001315dac/issues/gh/bloom-housing/bloom/4310)


## Description

History log on lottery page was showing every update to a listing that was already closed as a new closing. Now will only show actual listing closings, which can only occur after the listing was previously active.

## How Can This Be Tested/Reviewed?

For a lottery listing, close the listing and check it it appears in the history log. Make a change and save it to the listing, a new log should not be present. Reopen the listing, close the listing again, a new log should appear. All lottery related logs should still function as expected.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
